returnn/torch/engine.py

```diff
@@ -386,12 +386,13 @@ def train_epoch(self):
                 if not _has_data[0]:
                     break
 
+                # convert values from torch int32 to Python ints to prevent overflow
```

Comment:
Oh, good catch. This is not really related to this PR though, right? Maybe move it out and push separately directly to master?

Reply:
Ah, oh, yes!